eureka-common/src/main/java/edu/emory/cci/aiw/cvrg/eureka/common/comm/AppPropertiesModes.java

```diff
@@ -58,31 +58,31 @@ public AppPropertiesModes() {
 	 *
 	 * @return true or false.
 	 */
-	public boolean isDemoMode() {
```

Comment:
It's okay and recommended to use camelcase here. Just not in the application.properties.